packages/vscode-extension/src/extension.ts

```diff
@@ -430,6 +441,33 @@ function registerChatParticipant(context: vscode.ExtensionContext) {
   context.subscriptions.push(generateManifestGUID);
 }
 
+/**
+ * Copilot Chat Participant for Office Add-in
+ */
+function registerOfficeChatParticipant(context: vscode.ExtensionContext) {
+  const participant = vscode.chat.createChatParticipant(
+    officeChatParticipantId,
+    officeChatRequestHandler
+  );
+  participant.iconPath = vscode.Uri.joinPath(context.extensionUri, "media", "teams.png");
+  participant.followupProvider = followupProvider;
+  participant.onDidReceiveFeedback((e) => handleFeedback(e));
+
+  context.subscriptions.push(
+    participant,
+    vscode.commands.registerCommand(CHAT_CREATE_OFFICE_SAMPLE_COMMAND_ID, chatCreateCommandHandler),
```

Comment:
It's better to have commandId and handler 1:1 mapping. So, I think it's better to use a different handler or reuse the same command id.